<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>965</width>
    <height>745</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>965</width>
    <height>745</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>965</width>
    <height>745</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>聊天室</string>
  </property>
  <widget class="QListWidget" name="contentListWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>681</width>
     <height>411</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="inputTextEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>681</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="sendTextPushButton">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>690</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>发送文本</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendImagePushButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>690</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>发送图片</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitPushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>690</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QListWidget" name="noticeListWidget">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>40</y>
     <width>231</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="onlineListWidget">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>350</y>
     <width>231</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>10</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>公告</string>
   </property>
  </widget>
  <widget class="QLabel" name="onlineLabel">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>320</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>当前在线人数(0人)</string>
   </property>
  </widget>
  <widget class="QLabel" name="userLabel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>聊天内容</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
